src/Microsoft.VisualStudio.Editor.Razor/DefaultVisualStudioRazorParser.cs

```diff
@@ -180,7 +180,16 @@ internal void StartParser()
             Debug.Assert(_projectEngine.Engine != null);
             Debug.Assert(_projectEngine.FileSystem != null);
 
-            var projectDirectory = Path.GetDirectoryName(_documentTracker.ProjectPath);
+            string projectDirectory;
+            if (_documentTracker.ProjectPath != null)
+            {
+                projectDirectory = Path.GetDirectoryName(_documentTracker.ProjectPath);
+            }
+            else
+            {
+                projectDirectory = "/";
```

Comment:
How does this behave? Like... why does `/` work?

Reply:
Oh ya, this is totally a untested lol; just put this here and got out a design PR quickly because I had questionable feelings about the direction 😄.

Wasn't super concerned about this piece of code either since it's mostly temporary.

Reply:
Ok, so for a design PR I don't have any real concerns about this - but we do need to figure out how it should behave in this case.